src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/OffenderSentenceProfileCalculation.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
+
+import java.time.LocalDate
+
+data class OffenderSentenceProfileCalculation(
+  var licenceExpiryDate: LocalDate?,
+  var sentenceExpiryDate: LocalDate?,
+  var releaseDate: LocalDate?,
+  var topUpSupervisionDate: LocalDate?,
```

Comment:
is this the same thing as the _post sentence supervision expiry date_?

Reply:
It is :)

Reply:
We should ask louise her preference